charts/nexus-iq/values.yaml

```diff
@@ -19,9 +19,27 @@ iq:
   # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
   # licenseFile: /etc/nexus-iq-license/license_lic
   extraLabels:
+  # add the following two lines to mount a secrets volume within the container at the specified location  
+  secretName: #secret-jks
+  secretMountName: #/etc/secret-volume
```

Comment:
Is the `#` supposed to be there or at the front of the line?

Reply:
> Is the `#` supposed to be there or at the front of the line?

They are part of the contract but deliberately left empty. Could just as easily comment out the whole line?

Reply:
commenting the whole line would be more consistent with other commented values in this file.